src/cml.js

```diff
@@ -257,77 +257,38 @@ class CML {
     return await getDriver(this).runnerToken();
   }
 
-  parseRunnerLog(opts = {}) {
+  async parseRunnerLog(opts = {}) {
     let { data } = opts;
-    if (!data) return;
+    if (!data) return [];
 
-    const date = new Date();
+    data = data.toString('utf8');
 
-    try {
-      data = data.toString('utf8');
-
-      let log = {
-        level: 'info',
-        date: date.toISOString(),
-        repo: this.repo
-      };
-
-      if (this.driver === GITHUB) {
-        const id = 'gh';
-        if (data.includes('Running job')) {
-          log.job = id;
-          log.status = 'job_started';
-        } else if (data.includes('completed with result')) {
-          log.job = id;
-          log.status = 'job_ended';
-          log.success = data.includes('Succeeded');
-        } else if (data.includes('Listening for Jobs')) {
-          log.status = 'ready';
-        }
+    const logs = [];
+    const entities = await getDriver(this).runnerParseLogEntities();
```

Comment:
Same here and all the `entity` below: maybe a bit more descriptive?

Reply:
Same as [here](https://github.com/iterative/cml/pull/1030#discussion_r893162936)